test/test_main.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 from unittest.mock import AsyncMock, patch, ANY
 
-import main
```

Comment:
3287 :: technically this line works fine as-is, but feels weird :p  
`import main` by itself seems to work fine without breaking anything?

Reply:
I simplified the import